spec/support/revision_shared_examples.rb

```diff
@@ -61,6 +61,11 @@
 
       expect do
         visit project_trash_path(current_project)
+
+        within '#trash' do
```

Comment:
A bit unrelated to the task but it makes this specific spec not dependent on the other specs that create records by counting the records in the trash instead of expecting it to be empty.